xenmgr/XenMgr/Connect/Xl.hs

```diff
@@ -201,22 +206,51 @@ unpause uuid = do
                 exitCode <- system ("xl unpause " ++ domid)
                 bailIfError exitCode "Error unpausing domain."
 
+getXlProcess :: Uuid -> IO String
+getXlProcess uuid = do
+    let conv_uuid = TT.replace "_" "-" (show uuid)
+    (ec,str_pid,_) <- readProcessWithExitCode "pgrep" ["-f", conv_uuid] ""
```

Comment:
Maybe just save the xl pid when it is originally forked?

Reply:
Well it's not so much the expense, but more that xenmgr should keep track of its children.  However xl daemonizes, so it's not as simple as just tracking the spawned child.

Um, this just matches off the uuid, but there is no verification that it is an xl process.

Reply:
Good point, xl processes were the only ones I found on the platform that ever have the full uuid, but that may not be the case forever.